.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*.rs]
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Yes, but the thing is most vimmers use [rust.vim](https://github.com/rust-lang/rust.vim) and that configures vim appropriately in the Rusty way. It just seemed redundant to me, that's all... (Btw, I don't use editorconfig plugin for vim; in fact, it was after I saw this that I knew of its existence) It's your call...